<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferencesWindow</class>
 <widget class="QDialog" name="preferencesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/Anwalic_icon_1.png</normaloff>icon/Anwalic_icon_1.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>List Item Font Size</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboFontSize">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCustomBackground">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>128</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbSetCustomBackground">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>98</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Custom Background</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCustomBackground">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>128</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnApply">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>List Caption Color</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboListCaptionColor">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
